docs/using/index.html

```diff
@@ -576,6 +571,42 @@ <h1 id="customizing-tpots-operators-and-parameters">Customizing TPOT's operators
 <p>When using the command-line interface, the configuration file specified in the <code>-config</code> parameter <em>must</em> name its custom TPOT configuration <code>tpot_config</code>. Otherwise, TPOT will not be able to locate the configuration dictionary.</p>
 <p>For more detailed examples of how to customize TPOT's operator configuration, see the default configurations for <a href="https://github.com/rhiever/tpot/blob/master/tpot/config/classifier.py">classification</a> and <a href="https://github.com/rhiever/tpot/blob/master/tpot/config/regressor.py">regression</a> in TPOT's source code.</p>
 <p>Note that you must have all of the corresponding packages for the operators installed on your computer, otherwise TPOT will not be able to use them. For example, if XGBoost is not installed on your computer, then TPOT will simply not import nor use XGBoost in the pipelines it considers.</p>
+<h1 id="customizing-tpots-starting-population">Customizing TPOT's starting population</h1>
+<p>TPOT allows for the initial population of pipelines to be seeded. This can be done either through the <code>population_seeds</code> parameter in the TPOT constructor, or through a <code>population_seeds</code> attribute in a custom config file.</p>
+<pre><code class="Python">population_seeds = [
+    'BernoulliNB(GaussianNB(input_matrix), BernoulliNB__alpha=0.1, BernoulliNB__fit_prior=False)',
+    'BernoulliNB(input_matrix, BernoulliNB__alpha=0.01, BernoulliNB__fit_prior=True)'
```

Comment:
>How easy would it be to take actual sklearn pipelines as input instead of the string representations?

We don't have any code to go from sklearn pipelines to deap pipelines, only the other direction. It would be non-trivial to write, but doable given Python's complete reflection of objects.

>we should use better example clfs in the docs: Maybe a RandomForestClassifier and a LogisticRegression?

Sure, that's an easy enough change.